docs/source/build_app/build-application.rst

```diff
@@ -154,3 +164,9 @@ Static linking, Single-threaded |short_name|:
 
      clang++ -fsycl -DONEAPI_DAAL_USE_MKL_GPU_GEMM my_first_daal_program.cpp -Wl,
      --start-group <install dir>/daal/latest/lib/intel64/libonedal_core.a <install dir>/daal/latest/lib/intel64/libonedal_sequential.a -lpthread -ldl -lOpenCL -foffload-static-lib=<install dir>/daal/latest/lib/intel64/libonedal_sycl.a -Wl,--end-group
+
+Dynamic library versions
+---------------------
```

Comment:
we also provide this versions as part of env scripts - should be worth mentioning.

Reply:
Mention was added